src/character/gear/Gear.tsx

```diff
@@ -1,12 +1,75 @@
+import styled, { css } from "styled-components";
 import Page from "../../shared/page/Page";
 import { Locations } from "../../utils/Location";
 import { Money } from "./Money";
+import EditableItemEntry from "./EditableItemEntry";
+import { useContext } from "react";
+import { CharacterContext } from "../../DeadlandsCompanion";
+import { Icon } from "../../icons/Icon";
+import { Icons } from "../../icons/Icons";
+import { Theme } from "../../Theme";
+import { CustomItemData } from "../../utils/types/CustomItem";
+import { push, ref } from "firebase/database";
+import { database } from "../../utils/firebase/Firebase";
```

Comment:
Unused imports.

It might be worth checking that your ESLint is picking up the rules defined in the eslintrc, it should be giving you an error for this.

Reply:
I'm too used to having ESLint as part of our CI haha, we run and commit `npm run lint --fix` in CI 😄

Reply:
Fancy Schmancy ;)